index.html

```diff
@@ -1094,6 +1128,25 @@
 See [[fingerprinting-guidance]] for how to mitigate threats that result from
 [=fingerprinting=].
 
+Supercookies occur when a UA stores data for a site but makes that data more
+difficult to clear than other cookies or storage, typically because of a bug or
+because the browser restores the browser vendor's cookies when local state is cleared.
```

Comment:
This text is new, and I don't think it captures how supercookies typically happen. E.g. none of the supercookies described at https://blog.mozilla.org/security/2021/01/26/supercookie-protections/ or https://www.makeuseof.com/tag/what-are-supercookies-and-why-are-they-dangerous/ are in either of your categories.

Reply:
I added cache/network state info from the Fx post. The bug part comes from Unsanctioned Tracking. We have what MakeUseOf describes but under header enrichment instead. I think it makes sense to keep it separate because it's not something that the UA can do anything about.